oak_functions_containers_app/Cargo.toml

```diff
@@ -5,8 +5,14 @@ edition = "2021"
 license = "Apache-2.0"
 
 [features]
-default = ["native"]
+default = ["native", "deny_sensitive_logging"]
 native = ["dep:libloading", "dep:tempfile", "dep:ouroboros"]
+deny_sensitive_logging = [
+  "oak_functions_service/deny_sensitive_logging"
+]
+# Feature allow_sensitive_logging is not actually used in the code. It is only used as a
+# required feature to differentiate between the two binaries.
+allow_sensitive_logging = []
```

Comment:
I am pretty sure this will bite us back at some point, since features are supposed to be allowed to co-exist (see https://doc.rust-lang.org/cargo/reference/features.html#mutually-exclusive-features ).

Instead personally I would suggest creating two main.rs files (one per binary) and propagating a const / generic / something all the way down from there.

Reply:
I'll leave the cleanup to follow-up PRs :)

That'd apply to the Restricted Kernel version as well.